courses/machine_learning/deepdive2/image_classification/solutions/3_tf_hub_transfer_learning.ipynb

```diff
@@ -0,0 +1,750 @@
+{
```

Comment:
<p><a href="https://www.tensorflow.org/api_docs/python/tf/data/Dataset#cache" target="_blank">.cache</a>&nbsp;is key here. It will store the dataset in memory</p><p> </p><p>cool! will this only work for in-memory datasets?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Not sure if I'm understanding your question correctly, but if you give it a filepath, it will save the contents there instead of storing it in memory.</p><p>Does that answer your question?</p>